includes/definitions.inc.php

```diff
@@ -1435,6 +1435,19 @@
 $config['os'][$os]['over'][2]['graph'] = 'device_mempool';
 $config['os'][$os]['over'][2]['text']  = 'Memory Usage';
 
+// Lenovo EMC (NAS)
+$os = 'lenovoemc';
```

Comment:
Checked it (from the USB device / interface) ... it's Linux, 2.6.31.8. Is there a specific format / way to use this info? New to this, sorry if that's a dumb question.